### PR TITLE
Day4 bus mall styles

### PR DESCRIPTION
**Pull request by DocJek**
Added borders to heading, figure captions, and chart

### CODE REVIEW

**Comment by DocJek** [comment]
#6 
Added transitions and transform to images, added styling to fonts and borders to text and chart backgrounds.  Switched display settings to flexbox.

File: js/app.js [hunk 1]
```diff
@@ -26,6 +26,7 @@ function getElWithId (id) {
   console.log('Could not find id');
 };
 
+// actually renders built bar graph on page
```
Comment (Tyler-206) [inline js/app.js:29]: Good use of comments

File: index.html [hunk 2]
```diff
@@ -2,14 +2,15 @@
...
+    <link href="https://fonts.googleapis.com/css?family=Permanent+Marker" rel="stylesheet">
     <script src="https://cdnjs.cloudflare.com/ajax/libs/Chart.js/2.4.0/Chart.min.js"></script>
     <meta charset="utf-8">
     <title>BusMall Focus Group Survey</title>
   </head>
   <body>
     <header>
         <h1>Welcome to BusMall</h1>
-        <h2>Which of these would you be interested in purchasing?</h2>
+        <h2>Click on the product you be interested in purchasing, foo!</h2>
```
Comment (squeekyfoot) [inline index.html:13]: Good job, providing additional instructions to the user!

File: js/app.js [hunk 3]
```diff
@@ -113,6 +118,7 @@ function renderPic () {
     figcaption.textContent = productLineUp[tempRandom].name;
     img.setAttribute('src', 'images/' + productLineUp[tempRandom].path);
     img.setAttribute('id', productLineUp[tempRandom].name);
+    img.setAttribute('class', 'blur');
```
Comment (Tyler-206) [inline js/app.js:121]: i like how you added attributes in js. I need to practice this!

File: index.html (hunk 2 above)
Comment (Tyler-206) [inline index.html:13]: better wording!

File: index.html [hunk 4]
```diff
@@ -2,14 +2,15 @@
 <html>
   <head>
     <link href="css/style.css" type="text/css" rel="stylesheet"/>
+    <link href="https://fonts.googleapis.com/css?family=Permanent+Marker" rel="stylesheet">
```
Comment (Tyler-206) [inline index.html:5]: google fonts is such a great tool!

File: css/style.css [hunk 5]
```diff
@@ -21,13 +34,29 @@ body {
 }
 
 canvas {
+  border: 2px solid blue;
   background-color: white;
 }
 
-#pic * {
-  display: inline;
+#pic {
```
Comment (Tyler-206) [inline css/style.css:41]: review comment

**Review by Tyler-206 — COMMENTED** [review]
review practice

**Pull request merged**